public/docs/ts/latest/quickstart.jade

```diff
@@ -1,39 +1,53 @@
-include _util-fns
-  
-:marked
-  Our QuickStart goal is to build and run a super-simple Angular 2 application in TypeScript.
+block includes
+  include _util-fns
+  - var _Install = 'Install'
+  - var _angular_browser_uri = '@angular/platform-browser-dynamic'
+  - var _angular_core_uri = '@angular/core'
+  - var _appDir = 'app'
+  - var _indexHtmlDir = 'project root'
+  - var quickstartSourceUri = 'https://github.com/angular/quickstart/blob/master/README.md'
 
-  # Download the code
-  In a hurry?
-  [Download the QuickStart source](https://github.com/angular/quickstart/blob/master/README.md)
-  and start coding.
-  
-  # See it run
-
-  Try this <a href="/resources/live-examples/quickstart/ts/plnkr.html" target="_blank">live example</a>
-  which loads the sample app in <a href="http://plnkr.co/" title="Plunker" target="_blank">plunker</a>
-  and displays a simple message:
+:marked
+  Our QuickStart goal is to build and run a super-simple 
+  Angular 2 application in #{_Lang}, and 
+  establish a development environment for the remaining documentation samples
+  that also can be the foundation for real world applications.
+
+.callout.is-helpful
+  header Don't want #{_Lang}?
+  p.
+    Although we're getting started in #{_Lang}, you can also write Angular 2 apps
+    in #{_docsFor == 'ts' ? 'Dart' : 'TypeScript'} and JavaScript.
+    Just select either of those languages from the combo-box in the banner.
+
+h1 Try it!
+p
+  | Try the #[+liveExampleLink2('live example', 'quickstart')] which loads the sample app
+  +ifDocsFor('ts')
+    | &nbsp;in #[a(href="http://plnkr.co/" title="Plunker" target="_blank") plunker]
+  | &nbsp;and displays the simple message:
 
 figure.image-display
   img(src='/resources/images/devguide/quickstart/my-first-app.png' alt="Output of QuickStart app")
 
+h1 Build this app!
 :marked
-  # Learn
-  Of course we don't build apps to run in plunker.
-  The following steps establish a development environment for the documentation samples
-  that also can be the foundation for our real world applications. At a high level, we will
+  - [Step 0](#set-up): set up.
+  - [Step 1](#create-and-configure): create the app’s project folder and
+    define package dependencies and special project setup.
+  - [Step 2](#root-component): create the app’s Angular root component.
+  - [Step 3](#main): add `main.#{_docsFor}`, identifying the root component to Angular.
+  - [Step 4](#index): add `index.html`, the web page that hosts the application.
```

Comment:
I think that this step title is too simple to just be "add index.html", because adding the file is like 30% of the work.